.gitignore

```diff
@@ -1,6 +1,8 @@
 node_modules
 .env
+.env.*
```

Comment:
If we do this, maybe we should also add a `!.env.example` below. It's not necessary, since that file is already in the git index, but it would make explicit that it shouldn't be ignored.